.travis.yml

```diff
@@ -1,6 +1,13 @@
 services:
   - docker
 
+script:
```

Comment:
let's not run the benchmarks during the CI

Reply:
I had some issues with C compilation locally and I wanted to test it on Travis. However, I plan to remove it before the merge.